Woof/Woof/Model/Rating/StarsRating.swift

```diff
@@ -0,0 +1,8 @@
+/// A 5-star rating system, where 5 stars represent the highest rating and 1 star represents the lowest rating.
+enum StarsRating: Equatable {
```

Comment:
do we need to mention the unavailable case as well?

Reply:
Done.